test/pokemon.c

```diff
@@ -307,3 +307,19 @@ TEST("givemon [vars]")
     EXPECT_EQ(GetMonData(&gPlayerParty[0], MON_DATA_GIGANTAMAX_FACTOR), TRUE);
     EXPECT_EQ(GetMonData(&gPlayerParty[0], MON_DATA_TERA_TYPE), TYPE_FIRE);
 }
+
+TEST("checkteratype/setteratype work")
```

Comment:
Is it worth it to parametrize all types?

Reply:
Maybe? Hyper Train doesn’t parametrise all stats, so I didn’t do it either

Reply:
Then it's fine. We can decide later if we need to parametrize it.